ToolBox/QuantConnect.ToolBox.csproj

```diff
@@ -36,7 +36,7 @@
     <LangVersion>6</LangVersion>
   </PropertyGroup>
   <PropertyGroup>
-    <StartupObject>QuantConnect.ToolBox.KaikoDataConverter.Program</StartupObject>
+    <StartupObject>QuantConnect.ToolBox.CoarseUniverseGenerator.Program</StartupObject>
```

Comment:
Not sure if this change was intentional or matters, don't think we have a 'default' startup program

Reply:
`QuantConnect.ToolBox` is the only project in the LEAN solution with multiple `Main` methods (programs) and VS stores the currently selected startup object/program in the `StartupObject` tag of the `csproj` file. 

Although there is a `(Not set)` option in the VS project properties for this tag, selecting it will cause this build error:
```
error CS0017: Program has more than one entry point defined. Compile with /main to specify the type that contains the entry point.
```
I think in the past we have always set this value to the last updated program in the project.

To get rid of this issue, I think we should move each program to its own project.
This would also be a nice thing, every program would not be compiled to the same `QuantConnect.ToolBox.exe` filename :)

Reply:
Or Toolbox could have a master project which can call the other more like class libraries; removing all main methods and concentrating it into one. 

`toolbox.exe --app=YahooDownloader SYMBOLS RESOLUTION FROMDATE TODATE`

Reply:
New issue #2065 for this